script/reset_database.sh

```diff
@@ -10,19 +12,14 @@ development) ;;
 esac
 
 bundle install
```

Comment:
With the changes you've made, it appears that this `bundle install` line is the only code shared between the two environments. Would it make sense to move all of the code into the case statement above?